charts/osm/README.md

```diff
@@ -101,9 +101,15 @@ The following table lists the configurable parameters of the osm chart and their
 | OpenServiceMesh.fluentBit.workspaceId | string | `""` | WorkspaceId for Fluent Bit output plugin to Log Analytics |
 | OpenServiceMesh.grafana.enableRemoteRendering | bool | `false` | Enable Remote Rendering in Grafana |
 | OpenServiceMesh.grafana.port | int | `3000` | Grafana service's port |
-| OpenServiceMesh.image.pullPolicy | string | `"IfNotPresent"` | Container image pull policy |
-| OpenServiceMesh.image.registry | string | `"openservicemesh"` | Container image registry |
-| OpenServiceMesh.image.tag | string | `"v0.9.2"` | Container image tag |
+| OpenServiceMesh.image.digest | object | `{"osmBootstrap":"","osmCRDs":"","osmController":"","osmInjector":"","osmSidecarInit":""}` | Image digest (defaults to latest compatible tag) |
+| OpenServiceMesh.image.digest.osmBootstrap | string | `""` | osm-boostrap's image digest |
```

Comment:
For readability - I wonder if instead of `OpenServiceMesh.image.digest.osmBootstrap` this should be `OpenServiceMesh.image.osmBootstrap.digest`  (i.e. swap the positions of `digest` and `osmBootstrap`)
This mainly because I see that the comment to this value is `osm-boostrap's image digest` (first name of image, then digest)

Reply:
That works, I felt it's easier to keep `image.digest` as the top-level key and use the image names as child keys, so we have:
```
image
  digest:
     foo: ..
     bar: ...
     baz: ...
```

instead of:
```
image:
  foo:
    digest: ...
  bar:
    digest: ...
  baz:
    digest: ...
```

Reply:
I'm not sure it makes much of a difference now, but if we decide later to encode more things for each image, like if we make it possible to pull different images from different repos and with different tags, then it definitely makes sense to reorganize the values to `image.<name>.digest`. But reorganizing the values like that would be a breaking change to the Helm chart which we should do our best to avoid.

Reply:
For simplicity, I don't think we should be allowing mixing tags for different images, that's an unnecessary complexity I wouldn't consider at the moment.